plugins/Ecommerce/Controller.php

```diff
@@ -97,10 +96,19 @@ public function getConversionsOverview()
     public function getEcommerceLog($fetch = false)
     {
         $saveGET = $_GET;
-        $_GET['segment'] = urlencode('visitEcommerceStatus!=none');
+        $originalQuery = $_SERVER['QUERY_STRING'];
+
+        if (!empty($_GET['segment'])) {
+            $_GET['segment'] = $_GET['segment'] . ';' . 'visitEcommerceStatus!=none';
+        } else {
+            $_GET['segment'] = 'visitEcommerceStatus!=none';
+        }
+        $_SERVER['QUERY_STRING'] = http_build_query($_GET);
```

Comment:
Need to overwrite QUERY_STRING, as well, as it is used here https://github.com/piwik/piwik/blob/3.x-dev/core/API/Request.php#L408-L415